sonic-net/sonic-buildimage
# [Mellanox] Update MFT to v4.21.0-100

Signed-off-by: Volodymyr Samotiy <volodymyrs@nvidia.com>



#### Why I did it
To update MFT package to the latest version.

#### How I did it
Updated ```MFT_VERSION``` & ```MFT_REVISION``` in ```platform/mellanox/mft.mk```.

#### How to verify it
* Build an image and deploy to the switch
* Check MFT version by ```dpkg -l | grep mft```
* Verify that all the SONiC services up and running
* Run regression testing using [tests](https://github.com/sonic-net/sonic-mgmt/tree/master/tests) from sonic-mgmt

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

[Mellanox] Update MFT to v4.21.0-100

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
